x-pack/test/alerting_api_integration/common/lib/get_test_alert_data.ts

```diff
@@ -9,7 +9,7 @@ export function getTestAlertData(overwrites = {}) {
     enabled: true,
     name: 'abc',
     alertTypeId: 'test.noop',
-    interval: '10s',
```

Comment:
I'm curious about the change in `interval` from `10s` to `1m`.  Does the increased latency here help cut down on "noisy" alerts generating more test es docs?

Reply:
Yeah, I mostly bumped it for protection if ever it would execute the alert again. I haven't seen that scenario happen myself but could see it if we're waiting > 10s for es docs. There was no reason behind the `10s` so I figured I might as well bump it to something less likely to run.